config/manager/manager.yaml

```diff
@@ -2,25 +2,25 @@ apiVersion: v1
 kind: Namespace
 metadata:
   labels:
-    control-plane: controller-manager
-  name: system
+    control-plane: ssp-controller-manager
+  name: kubevirt
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
-  namespace: system
+  name: ssp-controller-manager
```

Comment:
The deployment can have name `ssp-operator`. It would be clearer.
The labels can also be changed to `control-plane: ssp-operator`.
I'm not sure why the key is `control-plane` and if we can change it.

Reply:
I agree, I think the `control-plane` key is there to integrate with OLM in some way, not sure though